editions/de-AT-server/tiddlywiki.info

```diff
@@ -2,7 +2,8 @@
 	"description": "Server configuration of the German (Österreich) edition",
 	"plugins": [
 		"tiddlywiki/tiddlyweb",
-		"tiddlywiki/filesystem"
+		"tiddlywiki/filesystem",
+		"tiddlywiki/highlight"
```

Comment:
@pmario @araname what do you think of the suggestion to remove the language `*-server` editions from the repo?